keps/sig-node/4438-sidecar-restart-termination/README.md

```diff
@@ -350,9 +356,53 @@ For Beta and GA, add links to added tests together with links to k8s-triage for
 https://storage.googleapis.com/k8s-triage/index.html
 
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
--->
 
 - <test>: <link to test coverage>
+-->
+
+###### Existing tests
```

Comment:
@wojtek-t for the context: https://docs.google.com/document/d/14En0jesyZFrDG8Epn1mYcWB0dSZqCLv-HwtIdMuov0o/edit#heading=h.kklhxigdyh8